src/codeflare_sdk/cli/commands/list.py

```diff
@@ -16,15 +20,32 @@ def cli():
 
 @cli.command()
 @click.option("--namespace", type=str)
-@click.option("--all", is_flag=True)
 @click.pass_context
-def raycluster(ctx, namespace, all):
-    """List all rayclusters in a specified namespace"""
-    if all and namespace:
-        click.echo("--all and --namespace are mutually exclusive")
-        return
-    namespace = namespace or ctx.obj.current_namespace
-    if not all:
+def raycluster(ctx, namespace):
+    """
+    List all rayclusters in a specified namespace or
+    all namespaces if no namespace is given
+    """
+    if namespace:
         list_all_clusters(namespace)
         return
     list_clusters_all_namespaces()
+
+
+@cli.command()
+@click.pass_context
+@click.option("--cluster-name", "-c", type=str)
+@click.option("--namespace", "-n", type=str)
+@click.option("--no-ray", is_flag=True)
+def job(ctx, cluster_name, namespace, no_ray):
+    """
+    List all jobs in a specified RayCluster or in K8S cluster
```

Comment:
This is not exactly right. in both cases the jobs are being run on K8s clusters. However, they may be run either through the ray or mcad-kubernetes schedulers. In the ray case there will be a "raycluster" that we can reference to get job info. In the mcad-kubernetes case there is only pods that get deployed (as their own short lived "cluster"). 

So you may want to change this to just "List all jobs submitted".